config/environments/development.rb

```diff
@@ -44,4 +44,13 @@
   config.action_mailer.default_url_options = { host: "0.0.0.0:3000" }
 
   config.log_level = :debug
+
+  config.after_initialize do
```

Comment:
wouldn't it be better in a `config/initializers/bullet.rb` as per the convention with gems? It'd be much easier to discover IMO.

Reply:
Ah sorry, I meant to remove this from the branch before I pushed this PR...

Reply:
Removed.

Reply:
It was actually cool. Along the lines of the new PROFILE env var. I think it's worth adding it in a new PR.